api/v3/Engage/Signpetition.php

```diff
@@ -109,6 +109,9 @@ function _civicrm_api3_engage_signpetition_process($params) {
       CRM_Gpapi_Processor::addToGroup($contact_id, 'Community NL');
     }
 
+    //remove "Opt Out" and "do not email"
```

Comment:
Above you can see comment (GP-463: "der Group "Donation Info" Eintrag soll immer gesetzt werden..."). If contact is always added to group, that's why we always call 'enableSubscription' method.

Reply:
Will check this, my best guess is that we process "Donation Info" opt-in under a different legal basis here compared to `Newsletter.subscribe`, hence no explicit permission to clear suppression flags.